modules/claims_api/app/workers/claims_api/special_issue_updater.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'sentry_logging'
+
+module ClaimsApi
+  class SpecialIssueUpdater
+    include Sidekiq::Worker
+    include SentryLogging
+
+    def perform(user, special_issues, auto_established_claim: nil)
+      service = bgs_service(user).contention
+
+      contentions = service.find_contentions_by_ptcpnt_id(user.participant_id)
+
+      contention = contentions[:benefit_claims].select do |claim|
+                     claim[:clm_id] == auto_established_claim.evss_id.to_s
+                   end [0]
```

Comment:
I still need to handle the cause there are _no_ matches, but is it valid to assume that any matches will be to a _single_ contention?

Reply:
i dont think so? i thought one of our last discussions around all that was that each "disability" in the list of disabilities in the json payload could/would map to a different contention. i was planning on sending something like this in if i took the ticket that modifies the other background job to call this job:
```
[
  { contention_id: ###, special_issues: [{special_issue_stuff_here}] }
]
```

Reply:
ahh ok, I assumed that a call could have multiple special issues, for for a single contention. I updated this logic now to handle multiple matches if that were the case